### ISSUE TITLE
network interface doesn't works on lxc 2.0

### ISSUE THREAD

**Issue by joenio** [lappis-tools/holodev#21]
for now the patch below solves the problem:

``` diff
diff --git a/holodev b/holodev
index 42e92df..10e3416 100755
--- a/holodev
+++ b/holodev
@@ -276,6 +276,8 @@ do_start() {
   sudo lxc-wait -n $CONTAINER_NAME -s RUNNING
   # be sure network is up and running
   lxc_attach service networking start
+  lxc_attach ifconfig eth0 up
+  lxc_attach dhclient eth0
 }

 do_stop() {
```

But we need a way "more portable" to solve this. I suppose this solution will break on some environments.

**Comment by joenio** [comment]
this issue already related on Debian:
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=820428

**Comment by joenio** [comment]
remove the packages "lxcfs" and "libpam-cgfs" from host machine solves this issue.

**Comment by joenio** [comment]
fixed by commit 71b5a48b3928c80edb19b5fd73c3e4e1cb8480a7
